src/packs/dependents.rs

```diff
@@ -0,0 +1,120 @@
+use std::collections::HashMap;
+
+use super::Configuration;
+
+type PackName = String;
+type ViolationType = String;
+type ViolationCount = usize;
+
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct Dependents {
+    pub public_dependents: Vec<PackName>,
```

Comment:
I think we might be conflating privacy and dependency checking here. This isn't about whether something is public or not, but whether the other pack has explicitly listed the pack they use as a dependency. If they haven't listed it, it's still a dependency, but an implicit one (in violation).

What do you think of calling these `explicit` and `implicit` dependencies/dependents?

Reply:
Great idea!